assets/src/apps/authoring/components/Flowchart/toolbar/FlowchartHeaderNav.tsx

```diff
@@ -167,7 +167,7 @@ export const FlowchartHeaderNav: React.FC<HeaderNavProps> = () => {
     const newPartData = {
       id: `${copiedPart.type}-${guid()}`,
       type: copiedPart.type,
-      custom: copiedPart.custom,
+      custom: { ...copiedPart.custom, x: copiedPart.custom.x + 20, y: copiedPart.custom.y + 20 },
```

Comment:
same here, should have a variable name for the 20 number, maybe even some kind of context configuration for the authoring of "pasteOffset" or something to be shared with all of these...